pkg/controller/configmap/config.go

```diff
@@ -129,15 +129,25 @@ func (adaptor *ConfigMapK8s) validateConfigMap(configmap *corev1.ConfigMap) []er
 	if err != nil {
 		appendErrorIfNotNil(&errs, errors.Wrapf(err, "mtu is invalid"))
 	}
+	// Check DEFAULT section log_file option
+	if cfg.Section("DEFAULT").Key("log_file").Value() != "" {
+		log.Info(fmt.Sprintf("DEFAULT section option: log_file is not allowed to change, current value:%s is reverted back to default value:None",
+			cfg.Section("DEFAULT").Key("log_file").Value()))
+		// Revert DEFAULT log_file to default value
+		fillDefault(cfg, "DEFAULT", "log_file", "", true)
+		// Write config back to ConfigMap data
+		(*data)[operatortypes.ConfigMapDataKey], err = iniWriteToString(cfg)
```

Comment:
Do we need to reset the value call iniWriteToString if in any case we are failing validation and therefore setting operator in degraded state?

Reply:
if there is failing or error when calling iniWriteToString, we write error in NCP install CRD status, so, it means NCP install CRD is a degraded state. I feel this approach should be ok.